specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -9444,6 +9447,18 @@
         "primary": {
           "type": "boolean",
           "description": "Specifies the primary network interface in case the virtual machine has more than 1 network interface."
+        },
+        "deleteOption": {
+          "type": "string",
+          "description": "Specify what happens to the network interface when the VM is deleted",
+          "enum": [
+            "Delete",
+            "Detach"
+          ],
+          "x-ms-enum": {
+            "name": "DeleteOptions",
```

Comment:
Be aware that enums with the same name, even across multiple operations, must have the same values going forward at all times.

Reply:
@basharg